Makefile

```diff
@@ -2,32 +2,15 @@
 clean:
 	npx lerna exec "rm -rf node_modules dist build"
 	rm -rf node_modules
-	
-.PHONY: install
-install:
```

Comment:
Why are you removing these from the makefile?

Reply:
these are already in the package.json so I just removed the duplicates. Slowly moving to no makefile, like alex recommended